medusa/__main__.py

```diff
@@ -1094,6 +1090,10 @@ def initialize(self, console_logging=True):
                 # Disable flag to erase cache
                 app.SUBTITLES_ERASE_CACHE = 0
 
+            # Check if we need to perform a restore of the cache folder
+            Application.restore_cache_folder(app.CACHE_DIR)
```

Comment:
Keep these two lines where they we're. This is the restore feature. You only need the next line here IMO.

Reply:
If we restore before we could erase if the erase setting is enabled. Right?

Reply:
Yes. But, I mean.. in that case, what do you think the user wants to do?Restore the old cache or remove it?

Reply:
Oh, I see what you mean now...
I guess you're right :+1: